content/SCALE/SCALETutorials/Shares/SMB/AddSMBHomeShare.md

```diff
@@ -14,103 +14,159 @@ They are not recommended for new deployments.
+{{< include file="/static/includes/LocalSMBUser.md" >}}
 
-Starting in SCALE 24.04 (Dragonfish), the root filesystem of TrueNAS is read-only, which prevents `pam_mkhomdir` from creating the `/nonexistent` directory in cases where it previously did.
-This results in a permissions error if `pam_open_session()` is called by an application for a user account that has **Home Directory** set to **/nonexistent**.
-{{< /expand >}}
-{{< /hint >}}
+You can use an existing dataset for the share or create a new dataset.
+You can either add a share when you [create the dataset]({{< relref "DatasetsSCALE.md" >}}) for the share on the **Add Dataset** screen, or create the dataset when you add the share on the **Add SMB** screen.
+If creating a simple SMB share and dataset use either method, or if customizing the dataset, use the [**Add Dataset** screen]({{< relref "DatasetsSCALE.md" >}}) to access dataset advanced setting options.
+To configure a customized SMB share, use the **Add SMB** share option that provides access to the advanced setting options for shares.
+This procedure covers creating the share and dataset from the **Add Share** screen.
 
+To create an alternative to the legacy SMB home share:
 
-### Adding Share Users with Directory Services
+1. Go to **Shares**, click **Add** on the **Windows (SMB) Shares** widget to open the **Add SMB** screen.
 
-You can use Active Directory or LDAP to create share users.
+   If you created the dataset already, you can add the share with the correct share preset on this screen.
+   If you are creating the share and dataset together you can create both using the correct share preset on this screen.
 
-If not already created, add a pool, then join Active Directory.
+2. Browse to or enter the location of an existing dataset or path to where you want to create the dataset to populate the **Path** for the share.
+   To add a dataset, click **Create Dataset**, enter a name for the dataset, then click **Create Dataset**.
+   For example, creating a share and dataset named *private*.
+   
+   {{< trueimage src="/images/SCALE/Shares/AddPrivateSMBShare.png" alt="Set the Share Path" id="Set the Share Path" >}}
 
-Go to **Storage** and [create a pool]({{< relref "CreatePoolWizard.md" >}}).
+   Follow name naming conventions for:
+   * [Files and directories](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions)
+   * [Share names](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-fscc/dc9978d7-6299-4c5a-a22d-a039cdc716ea)
 
-Next, [set up the Active Directory]({{< relref "/SCALE/SCALETutorials/credentials/directoryservices/configadscale.md" >}}) that you want to share resources with over your network.
+   By default, the dataset name populates the share **Name** field and becomes the share name, and it updates the **Path** automatically.
```

Comment:
```suggestion
   By default, the dataset name populates the share **Name** field and updates the **Path** automatically.
```

Reply:
What this is doing or saying is by creating the dataset TrueNAS populates the name field thus naming the share the same as the dataset, if fails if you try to enter a new name other than the dataset. I can make this more clear so I'll reword this.